packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/modules/DevLauncherDevMenuExtensions.kt

```diff
@@ -32,7 +32,7 @@ class DevLauncherDevMenuExtensions(
         isEnabled = { true }
         label = { "Back to launcher" }
```

Comment:
```suggestion
        label = { "Back to Launcher" }
```

Title case to match casing of other dev menu items